bindings/action_system.json

```diff
@@ -54,6 +54,13 @@
           "deltas": "Deltas"
         },
         "return": "FinishedSpends"
+      },
+      "calculate_deltas": {
+        "type": "static_method",
```

Comment:
This should be `static` rather than `static_method`, and I think it should be on the `Deltas` object. So like `Deltas.fromActions(actions)`